app/utils/HttpResponseHelper.scala

```diff
@@ -37,7 +39,41 @@ trait HttpResponseHelper extends HttpErrorFunctions {
         throw new UnrecognisedHttpResponseException(methodContext, url, response)
     }
 
+  def parseJson(json: String, method: String, url: String): JsValue = {
```

Comment:
There are no unit tests for this method. Could make private and then write unit tests for validateJson or else leave public and write unit tests for both.